DFC.ServiceTaxonomy.Editor/Startup.cs

```diff
@@ -51,6 +53,8 @@ public void ConfigureServices(IServiceCollection services)
                 options.Cookie.Name = "staxantiforgery_Default";
             }), order:10);
 
+            services.Configure<MvcOptions>(options => options.Filters.Add(typeof(UnhandledExceptionFilterAttribute)));
```

Comment:
Perhaps this could be moved into `AddSlackMessagePublishing()` and called according to a passed in flag?